Add Full-Size Hackpad.V1

aryatajne28 opened the pull request:
Ready to submit your project? Nice job! Hopefully you've read the [submission guidelines](https://hackpad.hackclub.com/submitting). **If not, make sure to read them first.**

Check off the following to let the reviewers know you've done everything:
[X] I HAVE RAN DRC ON MY DESIGN WITH ZERO ERRORS (Silkscreen errors are okay)
[X] I HAVE BRANDED ALL MY PARTS (PCB, CASE) WITH MY HACKPAD'S NAME
[X] I HAVE MADE SURE MY 3D PRINTED PARTS DO NOT REQUIRE SUPPORTS (Text branding overhang is okay)
[X] I HAVE MADE SURE TO INCLUDE TOLERANCES (0.25mm)
[X] MY HACKPAD'S PCB IS UNDER 100mm x 100mm, I DO NOT HAVE MORE THAN 16X SWITCHES, 2X ROTARY ENCODERS, 1X OLED, 16X LEDS.
[X] IN MY FOLDER, THERE IS A README.MD FILE THAT CONTAINS THE FOLLOWING INFORMATION:
- Full render including all the parts
- A few sentences about the inspiration and challenges
- BOM
- Photos of the Schematic, PCB, and case

If you're unsure about the format, make sure to check out this [example]([insertexample](https://github.com/hackclub/hackpad/tree/main/hackpads/Duccs%20Fidget%20Toy)) and follow the format there.

If this PR isn't a submission, that's okay too! Please remove the above boxes and explain what the pull request is for! If there's anything about this PR template that seems confusing, send a message in #hackpad in slack!

Gamer153 commented:
Hey, you seem to have forgotten the pictures of the schematics and PCB, and I couldn't find a picture of the full assembled hackpad with all parts.

Also, a lot of the other approved requests have the images included as files directly, not through some Github link, so I think it'd be good if you add the pictures directly to your PR and README as files.

aryatajne28 commented:
Thanks for this! On it right away!

aryatajne28 commented:
Hey @Gamer153, I have updated the images of the complete Hackpad and Schematic/PCB. I have added a folder of images separately, but while updating in README, it was still converting it into github links so I used CDN links from Slack. Hope this helps!

Gamer153 commented:
Looks good!
Btw, if your image is in the repo on GitHub, you can add it directly in Markdown, and don't have to drag it in: e.g. for the pcb, instead of the link you could put this in your README: `![](img/PCB.png)`

Also, according to [this slack thread](https://hackclub.slack.com/archives/C07LESGH0B0/p1741788219693979?thread_ts=1741787888.794879&cid=C07LESGH0B0), you should put branding on everything if you're not printing it yourself.

Gamer153 commented:
I approve of this submission! Here's my own approved PR link: https://github.com/hackclub/hackpad/pull/443

qcoral reviewed (approved):
Awesome job on this hackpad!! officially approved.!!!jhknfui842nf

fill out this form to get it:

https://forms.hackclub.com/hackpadv2

aryatajne28 commented:
Thank you so much!
